packages/editor/src/plugins/exercise/editor.tsx

```diff
@@ -97,19 +93,36 @@ export function ExerciseEditor(props: ExerciseProps) {
             <p className="mb-2 text-gray-400">
               {exTemplateStrings.addOptionalInteractiveEx}
             </p>
-            <div className="-ml-1.5 flex">
-              {interactiveExerciseTypes.map((type) => {
-                return (
-                  <AddButton
-                    key={type}
-                    onClick={() => interactive.create({ plugin: type })}
-                    secondary
-                    dataQa={`add-exercise-${type}`}
-                  >
-                    {exTemplateStrings[type]}
-                  </AddButton>
-                )
-              })}
+            <div className="flex items-start">
+              {interactiveExerciseMenuItems.map(
+                ({ pluginType, title, icon, description }, index) => {
+                  const tooltipClassName =
+                    index === 0
+                      ? 'left-0'
+                      : index + 1 < interactiveExerciseMenuItems.length
+                        ? '-left-24'
+                        : 'right-0'
```

Comment:
Minor: I'd slightly prefer having this as a helper, instead of in this mapper, just to clean up the JSX as much as possible.

```jsx
{interactiveExerciseMenuItems.map(
  ({ pluginType, title, icon, description }, index) => (
    <button
      key={title}
      data-qa={`add-exercise-${pluginType}`}
      onClick={() =>
        interactive.create({
          plugin: pluginType as InteractivePluginType,
        })
      }
      className="serlo-tooltip-trigger w-full rounded-md p-1 hover:shadow-xl focus:shadow-xl"
    >
      <EditorTooltip
        className={getTooltipPosition(
          interactiveExerciseMenuItems.length,
          index
        )}
        text={description}
      />
      {icon || <IconFallback />}
      <b className="mt-2 block text-sm">{title}</b>
    </button>
  )
)}
```

```typescript
function getTooltipPosition(length: number, index: number) {
  return index === 0 ? 'left-0' : index + 1 < length ? '-left-24' : 'right-0'
}
```